src/pages/Contents/Producers/Process.scss

```
@@ -0,0 +1,58 @@
+.process {
+  margin: 20px 0;
+
+  .title {
+    margin: 20px 20px;
+  }
+
+  .process-all {
+    margin: 10px 0 10px;
+
+    &::after {
+      content: '';
+      display: block;
+      margin: 0 auto;
+      width: 598px;
+      border-bottom: 1px solid #f0f0f0;
+    }
+
+    .producers-all {
+      display: flex;
+      flex-direction: column;
+      flex-wrap: wrap;
+      height: 250px;
+      overflow-x: scroll;
+
+      &::-webkit-scrollbar {
+        display: none;
+      }
+
+      .producer {
+        display: flex;
+        width: 275px;
+        margin: 9px 0 9px 18px;
+
+        .producer-image {
+          width: 55px;
+          margin-right: 10px;
+          border-radius: 5px;
+        }
+
+        .producer-info {
+          display: flex;
+          flex-direction: column;
+          justify-content: center;
+          margin-left: 3px;
+
+          .producer-name {
+            margin: 3px 0;
+          }
+
+          .producer-act {
+            margin: 3px 0;
+          }
```

```suggestion
          .producer-name, .producer-act  {
            margin: 3px 0;
          }
```
이렇게 사용하셔도 되고, 똑같은 값이라면 공통된 클래스명을 사용하셔도 됩니다!

---

감사합니다!